triggers.md

```diff
@@ -126,6 +129,45 @@ Currently the only requirement that is specified is denoted by `"anonymous-clien
 
 We require that user agents discard any rules with requirements that are not recognized or supported in the current configuration, so this system fails closed. Additionally, due to the conservative parsing rules, any UA which did not support requirements at all would discard all rules with requirements.
 
+### Window name targeting hints
+
+For implementation reasons, it can be difficult for user agents to prerender content without knowing which window it will end up in. For example, the browser might prerender content assuming that it will be activated in the current window, replacing the current content, but then the user might end up clicking on an `<a href="page.html" target="_blank">` link, so the prerendered content is wasted.
+
+To help with this, `"prerender"` rules can have a `"target_hint"` field, which contains a [valid browsing context name or keyword](https://html.spec.whatwg.org/#valid-browsing-context-name-or-keyword) indicating where the page expects the prerendered content to be activated. (The name "target" comes from the HTML `target=""` attribute on hyperlinks.)
+
+```html
+<script type=speculationrules>
+{
+  "prerender": [{
+    "source": "list",
+    "target_hint": "_blank",
+    "urls": ["page.html"]
+  }]
+}
+</script>
+<a target="_blank" href="page.html">click me</a>
+```
+
+This is just a hint, and is not binding on the implementation. Indeed, we hope that it one day becomes unnecessary, and all implementations can activate prerendered content into any target window. At that point, the field can be safely ignored, and removed from the specification. But at least for Chromium, getting to that point might take a year or so of engineering effort, so in the meantime `"target_hint"` gives developers a way to use prerendering in combination with new windows.
+
+Note that if a page is truly unsure whether a given URL will be prerendered into the current window or a new one, they could include prerendering rules for multiple target windows:
+
+```json
+{
+  "prerender": [
+   {"source": "list",
+    "target_hint": "_self",
+    "urls": ["page.html"]
+   },
+   {"source": "list",
+    "target_hint": "_blank",
+    "urls": ["page.html"]
+   }]
+}
+```
+
+However, in browsers such as Chromium that need the target hint, this will prerender the page twice, and thus use twice as many resources. So this is best avoided if possible.
```

Comment:
nit: I think technically "Chromium" is an open-source project, not a browser. Maybe "such as Chrome" or "in Chromium-based browsers".

Reply:
Nice catch; switching to "implementations".